### ISSUE TITLE
IfcConvert: No support for file schema encountered ()

### ISSUE THREAD

**Issue by brunopostle** [IfcOpenShell/IfcOpenShell#2473]
Some code committed in the last day or so breaks here. e9a063c (2nd October) is ok, 1d39413 (3rd October) is not.

This is the error:

```
$ IfcConvert junk.ifc junk.dae
IfcOpenShell IfcConvert 0.7.0 (OCC 7.5.0)
Log:
[Error] [2022-10-04 20:16:09] std::bad_array_new_length
[Error] [2022-10-04 20:16:09] No support for file schema encountered ()
[Error] [2022-10-04 20:16:09] Unable to parse input file 'junk.ifc'
```

If I revert to e9a063c, everything is fine:

```
$ IfcConvert junk.ifc junk.dae
IfcOpenShell IfcConvert 0.7.0 (OCC 7.5.0)
Scanning file...
Done scanning file   
Parsing input file took 0 seconds
```

This is the python error, note I also get a segfault:

```
$ python3
Python 3.10.7 (main, Sep  7 2022, 00:00:00) [GCC 11.3.1 20220421 (Red Hat 11.3.1-2)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ifcopenshell
>>> ifcopenshell.open("junk.ifc")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.10/site-packages/ifcopenshell/__init__.py", line 107, in open
    raise exc(msg)
ifcopenshell.Error: Unable to parse IFC SPF header
>>> 
Segmentation fault (core dumped)
```

There is nothing interesting in the stack trace:

```
                Stack trace of thread 55882:
                #0  0x0000563584ceb180 n/a (n/a + 0x0)
```

**Comment by aothms** [comment]
This should already be fixed in f68db3d3bee94892a8789363400c8cb3df737e1d

Sent from a mobile device. Excuse my brevity. Kind regards, Thomas

Op di 4 okt. 2022 21:50 schreef Bruno Postle ***@***.***>:

**Comment by brunopostle** [comment]
Thanks, closing as fixed in current git
